modules/hcp-dr-oadp-restore-new-mgmt.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disaster-recovery-oadp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dr-oadp-restore-new-mgmt_{context}"]
+= Restoring a hosted cluster into a new management cluster by using {oadp-short}
+
+You can restore the hosted cluster into a new management cluster by creating the `Restore` custom resource (CR).
+
+* If you are using an in-place update, the `InfraEnv` resource does not need spare nodes. You need to re-provision the worker nodes from the new management cluster.
+* If you are using a replace update, you need some spare nodes for the `InfraEnv` resource to deploy the worker nodes.
+
+[IMPORTANT]
+====
+After you back up your hosted cluster, you must destroy it to initiate the restoring process. To initiate node provisioning, you must back up workloads in the data plane before deleting the hosted cluster.
+====
+
+.Prerequisites
+
+* You configured the new management cluster to use {oadp-short}. The new management cluster must have the same Data Protection Application (DPA) as the management cluster that you backed up from so that the `Restore` CR can access the backup storage.
+* You configured the networking settings of the new management cluster to resolve the DNS of the hosted cluster.
+
+** The DNS of the host must resolve to the IP of both the new management cluster and the hosted cluster.
+** The hosted cluster must resolve to the IP of the new management cluster. 
+
+* You completed the steps in link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/clusters/cluster_mce_overview#remove-a-cluster-by-using-the-console[Removing a cluster by using the console] to delete your hosted cluster.
+* You completed the steps in link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/clusters/cluster_mce_overview#removing-a-cluster-from-management-in-special-cases[Removing remaining resources after removing a cluster].
+
+To monitor and observe the backup process, see "Observing the backup and restore process".
+
+[IMPORTANT]
+====
+Complete the following steps on the new management cluster that you are restoring the hosted cluster to, not on the management cluster that you created the backup from.
+====
+
+.Procedure
+
+. Create a YAML file that defines the `Restore` CR:
++
+.Example `restore-hosted-cluster.yaml` file
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: <restore_resource_name> # <1>
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - clusters
+  - clusters-hosted-0
+  - infraenv-0
```

Comment:
Should we have the notes detailing which namespace to include here like in the backup resource [1]?

[1] https://github.com/openshift/openshift-docs/pull/95105/files#diff-ce2742dd5699961c020d184eaa9abf197486952dda3dcc24fb4b0560617ea914R94-R97